Add instructions for Google Cloud DNS forwarding (#2320)
cloudflare/cloudflare-docs: products/1.1.1.1/src/content/setup-1.1.1.1/google-cloud.md

---
order:
pcx-content: how-to
---

# Google Cloud

> Google Cloud supports configuring [outbound server policy](https://cloud.google.com/dns/docs/server-policies-overview#dns-server-policy-out) within Cloud DNS. Policies are applied per Virtual Private Cloud (VPC) network, and will apply to all resources within that VPC network, including any existing virtual machines.

Reviewer: ```suggestion
Google Cloud supports configuring [outbound server policy](https://cloud.google.com/dns/docs/server-policies-overview#dns-server-policy-out) within Cloud DNS. Policies are applied per Virtual Private Cloud (VPC) network, and will affect all resources within that VPC network, including any existing virtual machines.
```

Reviewer: Not sure if that's the right word, but trying to get away from the same verb 2x in a sentence

<Aside type="note">

> If you're using [Cloudflare for Teams](https://developers.cloudflare.com/cloudflare-one/), you can use the assigned [locations](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/locations) to apply custom [DNS policies](https://developers.cloudflare.com/cloudflare-one/policies/filtering/dns-policies-builder) via Gateway.

Reviewer: ```suggestion
If you are using [Cloudflare for Teams](https://developers.cloudflare.com/cloudflare-one/), you can choose assigned [locations](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/locations) to apply custom [DNS policies](https://developers.cloudflare.com/cloudflare-one/policies/filtering/dns-policies-builder) via Gateway.
```

Reviewer: Same as above

</Aside>

To configure 1.1.1.1 for your Google Cloud VPC network(s):

1. Open the [Google Cloud Console](https://console.cloud.google.com).

> 1. Head to **Network Services** > **Cloud DNS** and click on [**DNS Server Policies**](https://console.cloud.google.com/net-services/dns/policies).

Reviewer: ```suggestion
1. Navigate to **Network Services** > **Cloud DNS** and click [**DNS Server Policies**](https://console.cloud.google.com/net-services/dns/policies).
```

1. Click **Create Policy**.

> 1. Provide a name for your Policy (such as `cloudflare-1-1-1-1`), and select the VPC network or networks that this policy applies to.

Reviewer: ```suggestion
1. Provide a name for your Policy (such as `cloudflare-1-1-1-1`) and select associated VPC network or networks.
```
